### PR TITLE
Style: image list item and section

### PR DESCRIPTION
**Pull request by jessicaroque8**
This PR does the following:
- Adds a pointer cursor to LinkWrapper
- Changes styling on the ImageListItem Molecule (smaller font size and larger weight on title and allows for rows and smaller widths)
- Makes the ImageListItem an optional link
- Adds optional internal link to the story of ImageListItem
- Adds display attribute to SectionBody Atom
- Adds external links to LinkWrapper and ImageListItem Molecule

Steps to Test:

- Run storybook.
- Check that a pointer cursor appears in the LinkWrapper Atom.
- Check that you like the way the ImageListItem Molecule appears in its story.
- Check that ImageListItem has an option to add a 'to' in the knobs.

### CODE REVIEW

File: src/molecules/list-item/image-list-item.js [hunk 1]
```diff
@@ -24,34 +27,43 @@ class ImageListItem extends Component {
...
+					<ListItemHeader mt={3}>
+						<ListItemTitle>{title}</ListItemTitle>
+						{subtitle && (
+							<Subtitle pb={1} color="#717171">
+								{subtitle}
+							</Subtitle>
+						)}
+					</ListItemHeader>
+					<ListItemBody>{children}</ListItemBody>
+					{text && this.renderButton(buttonProps, text)}
```
Comment (joshferrell) [inline src/molecules/list-item/image-list-item.js]: renderButton should not be inside of the link wrapper. This outputs the following html: 
```
  <a href="/something">
    <h2>title</h2>
    <a href="/something">I'm the button</a>
  </a>
```

According to the w3c spec [the anchor tag is defined as](http://www.w3.org/TR/html5/text-level-semantics.html#the-a-element): 
>  Transparent, but there must be no interactive content descendant.

File: src/atoms/list-item-content/list-item-content.js [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-import { space, themeGet, lineHeight } from 'styled-system';
+import { space, themeGet, lineHeight, width, flex } from 'styled-system';
 import { createSkeletonElement } from '@trainline/react-skeletor';
 import { flipOrder } from 'styles';
 import { fontSize } from 'utils';
 import { IconCircle } from 'atoms';
 
 const makeListItemTitle = styled.h3.attrs({
-	fontSize: [3, 4],
+	fontSize: [1, 2],
```
Comment (joshferrell) [inline src/atoms/list-item-content/list-item-content.js]: can you pass fontSize in as an optional prop?
```
fontSize: props => props.fontSize || [1, 2]
```

Comment (jessicaroque8) [reply src/atoms/list-item-content/list-item-content.js]: Done! Good idea.

File: src/molecules/list-item/image-list-item.js (hunk 1 above)
Comment (jessicaroque8) [reply src/molecules/list-item/image-list-item.js]: Good catch, this is fixed.

**Comment by jessicaroque8** [comment]
The changes you requested are made, @joshferrell.

**Pull request merged**